.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [ 'main' ]
```

Comment:
Could we include stable as well for push and pull requests?

Reply:
Done!